artemis-cli/src/main/java/org/apache/activemq/artemis/cli/commands/queue/UpdateQueue.java

```diff
@@ -31,7 +31,7 @@
    String name;
 
    @Option(name = "--deleteOnNoConsumers", description = "whether to delete when it's last consumers disconnects)")
-   Boolean deleteOnNoConsumers = null;
+   String deleteOnNoConsumers = null;
```

Comment:
Hence you are suggesting to have:
1) --deleteOnNoConsumers to set the property to true
2) --noDeleteOnNoConsumers to set the property to false
3) nothing, to not update the property 
right?